client/my-sites/site-settings/form-general.jsx

```diff
@@ -414,6 +431,8 @@ const connectComponent = connect(
 			siteIsJetpack,
 			siteSlug: getSelectedSiteSlug( state ),
 			supportsHolidaySnowOption: siteIsJetpack && isJetpackMinimumVersion( state, siteId, '4.0' ),
+			jetpackSettingsUISupported: siteSupportsJetpackSettingsUi( state, siteId ),
```

Comment:
The prop `jetpackSettingsUISupported` is never used in this file. It can be removed. We should bring `jetpackMasterbarSupported` here instead.